scripts/runner-env.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+// Update environment variables for all runners in the given environment
+
+const apiUrl = 'https://api.render.com/v1';
+
+if (!process.env.ENVIRONMENT) {
+    console.error('Please set the ENVIRONMENT environment variable');
```

Comment:
what about a `help()` function showing how to use the script?

Reply:
Fixed in 17b64d01